New Shopify Manual (#89)
myparcelnl/developer: src/nl/documentatie/14.shopify.md

## Introductie

> Met onze Shopify plugin is het mogelijk verschillende soorten MyParcel zendingen aanmaken en gebruik maken van meerdere vervoerders:

Reviewer: ```suggestion
Met onze Shopify plugin is het mogelijk verschillende soorten MyParcel zendingen **aanmaken** en **gebruik maken** van meerdere vervoerders:
```

aan te maken en gebruik te maken

De app wordt bij wijzigingen automatisch bijgewerkt, je hebt hier zelf geen onderhoud aan.

### Eenmalige handeling

<MPImg src="/documentation/shopify/nl/shopify-api-key.jpg" alt="Shopify MyParcel API setting" />

> Voor het in gebruik nemen van de MyParcel app, moet er eerst een API key worden ingevuld. De API key is gratis te verkrijgen via <a href="https://backoffice.myparcel.nl/registration" target="_blank"> Shopinstellingen > API instellingen in de MyParcel Backoffice</a>.

Reviewer: ```suggestion
Voor het in gebruik nemen van de MyParcel app, moet er eerst een API key worden ingevuld. De API key is gratis te verkrijgen via <a href="https://backoffice.myparcel.nl/registration" target="_blank"> Shopinstellingen > API instellingen in de MyParcel Backoffice</a>.
```

Om de MyParcel App in gebruik te nemen, moet er...

> Na het invullen van de API key wordt de MyParcel app gekoppeld aan jouw Shopify shop.

Reviewer: ```suggestion
Na het invullen van de API key wordt de MyParcel app gekoppeld aan jouw Shopify shop.
```
De MyParcel app wordt gekoppeld aan jouw Shopify shop wanneer de API key is ingevuld.

### Algemene instellingen

<MPImg src="/documentation/shopify/nl/shopify-general-settings.jpg" alt="Shopify general settings" />

#### Standaard paginakeuze

Hier geef je aan of je het label op een standaard (A4) of op een labelprinter (A6) wilt printen.

#### Beschrijving op label

<MPImg src="/documentation/shopify/shopify-label-description.jpg" alt="Shopify label description" />

Met deze optie is het mogelijk een beschrijving toevoegen aan de zending. Dit zal worden afgedrukt op de linkerbovenhoek van het label en hiermee is het mogelijk om de zending in het MyParcel zendingenoverzicht op te zoeken. Gebruik `{order_name}` om de titel van de order of `{order_number}` om het ordernummer te vermelden.

#### Fulfillment

<MPImg src="/documentation/shopify/nl/shopify-fulfillment.jpg" alt="Shopify fulfillment" />

- Er is een optie om de bestellingen manueel / handmatig de status 'fulfilled' te geven. Ga naar Bestellingen, selecteer de gewenste orders en klik bovenaan het orderoverzicht op Bestellingen vervullen om deze status te veranderen.
- Je hebt de mogelijkheid om jouw orders automatisch de status 'Fulfilled' te geven, wanneer PostNL jouw zending heeft gescand.

> - Een andere optie is bij het printen / downloaden van een label de status op 'fulfilled' wordt gezet.

Reviewer: ```suggestion
- Een andere optie is bij het printen / downloaden van een label de status op 'fulfilled' wordt gezet.
```

"wordt gezet" vervangen door "te zetten"

### Account

### Standaard exportinstellingen

Aan de hand van de zones en verzendmethoden die in Shopify > Instellingen > Verzending en bezorging zijn gedefinieerd kunnen pakkettype, vervoerder en eventueel extra opties worden geconfigureerd.
Met de standaard exportinstellingen heb je de mogelijkheid om aan de hand van de door jouw klant gekozen verzendmethode automatisch een gewenst pakkettype zoals een pakket of een brievenbuspakje aan te maken. De door jou aangemaakte verzendmethoden worden hier direct ingeladen, zodat je gelijk aan de slag kunt met het instellen van de exportvoorkeuren. Heb je nog geen verzendmethoden aangemaakt? Bekijk dan de [handleiding van Shopify] hoe je dit kunt instellen.

<Stack class="flex">
    <iframe
        class="rounded-xl"
        width="560"
        height="315"
        src="https://www.youtube.com/embed/NyD-9CtW1q0"
        title="Geen verzendadres gevonden" allowfullscreen />
</Stack>

Wanneer je op een verzendmethode klikt, opent een pop-up. In de pop-up moet de verzendmethode worden geconfigureerd. Hier selecteer je:

- of de zending met MyParcel verzonden moet worden

> - met welke vervoerder

Reviewer: ```suggestion
- met welke vervoerder
```

met welke vervoerder de zending verzonden moet worden

Om de MyParcel App in gebruik te nemen, moet er eerst een API key worden ingevuld. De API key is gratis te verkrijgen via <a href="https://backoffice.myparcel.nl/registration" target="_blank"> Shopinstellingen > API instellingen in de MyParcel Backoffice</a>.
De MyParcel app wordt gekoppeld aan jouw Shopify shop wanneer de API key is ingevuld.

## Configuratie

### Algemene instellingen

<MPImg src="/documentation/shopify/nl/shopify-general-settings.jpg" alt="Shopify general settings" />

#### Standaard paginakeuze

Hier geef je aan of je het label op een standaard (A4) of op een labelprinter (A6) wilt printen.

#### Beschrijving op label

<MPImg src="/documentation/shopify/shopify-label-description.jpg" alt="Shopify label description" />

Met deze optie is het mogelijk een beschrijving toevoegen aan de zending. Dit zal worden afgedrukt op de linkerbovenhoek van het label en hiermee is het mogelijk om de zending in het MyParcel zendingenoverzicht op te zoeken. Gebruik `{order_name}` om de titel van de order of `{order_number}` om het ordernummer te vermelden.

#### Fulfillment

<MPImg src="/documentation/shopify/nl/shopify-fulfillment.jpg" alt="Shopify fulfillment" />

> - Er is een optie om de bestellingen manueel / handmatig de status 'fulfilled' te geven. Ga naar Bestellingen, selecteer de gewenste orders en klik bovenaan het orderoverzicht op Bestellingen vervullen om deze status te veranderen.

Reviewer 2: ```suggestion
- Er is een optie om de bestellingen manueel / handmatig de status "fulfilled" te geven. Ga naar "Bestellingen", selecteer de gewenste orders en klik bovenaan het orderoverzicht op "Bestellingen vervullen" om deze status te veranderen.
```

> - Je hebt de mogelijkheid om jouw orders automatisch de status 'Fulfilled' te geven, wanneer PostNL jouw zending heeft gescand.

Reviewer 2: Hier staat 'fulfilled' stelselmatig tussen enkele aanhalingstekens, maar verderop worden dubbele aanhalingstekens gebruikt voor letterlijke termen. Ik zou het in ieder geval binnen 1 handleiding consequent houden, en misschien over heel MyParcel zelfs.

- Ongefrankeerd label
- Digitale postzegel

**Beschikbare opties (mede afhankelijk van gekozen vervoerder):**

- (DHL Vandaag)
- Handtekening voor ontvangst
- Alleen geadresseerde
- Retour bij geen gehoor
- Extra groot formaat
- 18+-check
- Verzekering tot €&nbsp;5.000,–

::: note

> Let op! Bij de keuze voor DHL For You (Today) en DHL Vandaag moet er ook een afsluittijd worden meegegeven. Een label moet worden aangemaakt, het voor de afsluittijd aangemaakt is dan sturen we het mee.

Reviewer 3: ```suggestion
Let op! Bij de keuze voor DHL For You (Today) en DHL Vandaag moet er ook een afsluittijd worden meegegeven. Wanneer een label voor de afsluittijd aangemaakt is, sturen we het pakket mee.
```

Wanneer je de bestelling als een ander pakkettype wilt exporteren dan de eerder ingestelde standaard exportinstellingen, kan je dit binnen de MyParcel app doen. Ga naar `Apps > MyParcel NL` en open de gewenste order. Hier vind je de mogelijkheid om een export te maken met een aangepast pakkettype en eventuele verzendopties toevoegen of verwijderen.

<MPImg src="/documentation/shopify/nl/shopify-wijzig-verzending.jpg" alt="Shopify api settings" />

### Checkout afhaallocatie

Wanneer je [de optie 'afhaallocatie' aan een verzendmethode hebt gekoppeld](#standaard-exportinstellingen) hebben jouw klanten de mogelijkheid om op de bedankpagina na de betaling een afhaallocatie te selecteren. Wil je jouw klanten een herinnering sturen wanneer ze vergeten om het postpunt te selecteren? Zet dan de optie ['Herinneringsmail afhaallocatie'](#herinneringsmail-afhaallocatie) aan.

<MPImg src="/documentation/shopify/shopify-checkout-pickup.jpg" alt="Shopify checkout pickup" />

### Consument is vergeten een afhaalpunt te selecteren

<MPImg src="/documentation/shopify/nl/shopify-meer-acties.jpg" alt="Shopify bekijk bestelling status pagina" />

> Het is mogelijk om vanuit de shopify bestellingen overzicht naar de checkout pagina te gaan van een specifieke bestelling en hier alsnog een afhaallocatie te selecten. Ga naar Shopify Bestellingen overzicht > Selecteer de bestelling waar geen afhaalpunt geselecteerd is. Rechtsboven onder Meer acties > Bekijk de bestellingstatuspagina. Vervolgens is het mogelijk om een afhaalpunt te selecteren (zie afbeelding hieronder).

Reviewer: ```suggestion
Het is mogelijk om vanuit de shopify bestellingen overzicht naar de checkout pagina te gaan van een specifieke bestelling en hier alsnog een afhaallocatie te **selecten**. Ga naar Shopify Bestellingen overzicht > Selecteer de bestelling waar geen afhaalpunt geselecteerd is. Rechtsboven onder Meer acties > Bekijk de bestellingstatuspagina. Vervolgens is het mogelijk om een afhaalpunt te selecteren (zie afbeelding hieronder).
```
selecteren ipv selecten

<MPImg src="/documentation/shopify/nl/shopify-bestelling-status-pagina.jpg" alt="Shopify selecteer afhaalpunt" />

### Retourlabel mailen

> Heeft jouw klant een bestelling geplaatst en wil hij/zij deze retour sturen? Verstuur eenvoudig een mail vanuit Shopify met het retourlabel. Optie 1 vanuit de Shopify bestellingenoverzicht open je de order. Onder meer het Meer acties dropdownmenu vind je de optie 'Retourlabel versturen' om een mail naar jouw klant te sturen. Optie 2 is vanuit het MyParcel bestellingenoverzicht selecteer je een specifieke order en bovenaan de pagina vervolgens op de knop 'Verstuur retourlabel' klikken.

Reviewer 3: ```suggestion
Heeft jouw klant een bestelling geplaatst en wil hij/zij deze retour sturen? Verstuur eenvoudig een mail vanuit Shopify met het retourlabel. Optie 1: Vanuit het Shopify bestellingenoverzicht open je de order. Onder het 'Meer acties' dropdownmenu vind je de optie 'Retourlabel versturen' om een mail naar jouw klant te sturen. Optie 2: Vanuit het MyParcel bestellingenoverzicht selecteer je een specifieke order en bovenaan de pagina klik je vervolgens op de knop 'Verstuur retourlabel'.
```

Wil je instellingen zoals de opmaak van de mail en eigen bijdrage van jouw klant aanpassen? Dit kan in jouw [MyParcel backoffice]. Klik links onderin op `Shop instellingen > Selecteer de gewenste shop > Retour`.

### Testen

> Wij adviseren om, na het installeren en configureren van de plugin, het bestelproces te doorlopen door een test-bestelling te plaatsen en zo te bekijken of de checkout naar wens is. Meer informatie over het opzetten van een Shopify test omgeving: <a href="https://help.shopify.com/en/manual/checkout-settings/test-orders" target="_blank"> https://help.shopify.com/en/manual/checkout-settings/test-orders </a>

Reviewer: ```suggestion
Wij adviseren om, n**a het installeren en configureren van de plugin**, het bestelproces te doorlopen door een test-bestelling te plaatsen en zo te bekijken of de checkout naar wens is. Meer informatie over het opzetten van een Shopify test omgeving: <a href="https://help.shopify.com/en/manual/checkout-settings/test-orders" target="_blank"> https://help.shopify.com/en/manual/checkout-settings/test-orders </a>
```

bijzin vermijden

<MPImg src="/documentation/shopify/shopify-checkout-pickup.jpg" alt="Shopify checkout pickup" />

### Consument is vergeten een afhaalpunt te selecteren

<MPImg src="/documentation/shopify/nl/shopify-meer-acties.jpg" alt="Shopify bekijk bestelling status pagina" />

Het is mogelijk om vanuit het shopify bestellingen overzicht naar de checkout pagina te gaan van een specifieke bestelling en hier alsnog een afhaallocatie te selecteren. Ga naar Shopify Bestellingen overzicht > Selecteer de bestelling waar geen afhaalpunt geselecteerd is. Rechtsboven onder Meer acties > Bekijk de bestellingstatuspagina. Vervolgens is het mogelijk om een afhaalpunt te selecteren (zie afbeelding hieronder).

<MPImg src="/documentation/shopify/nl/shopify-bestelling-status-pagina.jpg" alt="Shopify selecteer afhaalpunt" />

### Retourlabel mailen

> Heeft jouw klant een bestelling geplaatst en wil hij/zij deze retour sturen? Verstuur eenvoudig een mail vanuit Shopify met het retourlabel. Optie 1: Vanuit het Shopify bestellingenoverzicht open je de order. Onder het 'Meer acties' dropdownmenu vind je de optie 'Retourlabel versturen' om een mail naar jouw klant te sturen. Optie 2: Vanuit het MyParcel bestellingenoverzicht selecteer je een specifieke order en bovenaan de pagina klik je vervolgens op de knop 'Verstuur retourlabel'.

Reviewer 2: Hier weer enkele aanhalingstekens voor letterlijke tekstjes van de knoppen, deze evt ook dubbel maken of eerder de dubbelen weer enkel maken.

<MPImg src="/documentation/shopify/nl/shopify-meer-acties.jpg" alt="Shopify bekijk bestelling status pagina" />

Het is mogelijk om vanuit het shopify bestellingen overzicht naar de checkout pagina te gaan van een specifieke bestelling en hier alsnog een afhaallocatie te selecteren. Ga naar Shopify Bestellingen overzicht > Selecteer de bestelling waar geen afhaalpunt geselecteerd is. Rechtsboven onder Meer acties > Bekijk de bestellingstatuspagina. Vervolgens is het mogelijk om een afhaalpunt te selecteren (zie afbeelding hieronder).

<MPImg src="/documentation/shopify/nl/shopify-bestelling-status-pagina.jpg" alt="Shopify selecteer afhaalpunt" />

### Retourlabel mailen

Heeft jouw klant een bestelling geplaatst en wil hij/zij deze retour sturen? Verstuur eenvoudig een mail vanuit Shopify met het retourlabel.

- Optie 1: Vanuit het Shopify bestellingenoverzicht open je de order. Onder het 'Meer acties' dropdownmenu vind je de optie 'Retourlabel versturen' om een mail naar jouw klant te sturen.
- Optie 2: Vanuit het MyParcel bestellingenoverzicht selecteer je een specifieke order en bovenaan de pagina klik je vervolgens op de knop 'Verstuur retourlabel'.

> - Optie 1: Vanuit het Shopify bestellingenoverzicht open je de order. Onder het 'Meer acties' dropdownmenu vind je de optie 'Retourlabel versturen' om een mail naar jouw klant te sturen.

Reviewer 2: ```suggestion
```

> - Optie 2: Vanuit het MyParcel bestellingenoverzicht selecteer je een specifieke order en bovenaan de pagina klik je vervolgens op de knop 'Verstuur retourlabel'.

Reviewer 2: ```suggestion
```
